assets/inputs/fqtk-samplesheet.csv

```diff
@@ -0,0 +1,2 @@
+id,samplesheet,lane,flowcell,per_flowcell_manifest
+test,https://github.com/fulcrumgenomics/nf-core-test-datasets/raw/fqtk/testdata/sim-data/fqtk_samplesheet.tsv,,https://github.com/nf-core/test-datasets/raw/demultiplex/testdata/sim-data/fastq.tar.gz,https://raw.githubusercontent.com/fulcrumgenomics/nf-core-test-datasets/fqtk/testdata/sim-data/per_flowcell_manifest.csv
```

Comment:
TODO @sam-white04: Add code to change csv to tsv, and make PR to change this in nf-core/test-datasets